Epsilon.Host.Frontend/src/views/CompetenceDocument.vue

```diff
@@ -1,3 +1,11 @@
 <template>
```

Comment:
Just beware that by using this store approach it becomes very difficult to see where data is coming from. I don't think the components itself should incorporate the store variables, instead they can be used in the properties, making it clear where the data is coming from and allowing for more scalability in case we switch from store type.

Reply:
Well oke... But i need to export to mention it in `main.ts`